src/core/mac/mac.cpp

```diff
@@ -165,6 +165,11 @@ Mac::Mac(ThreadNetif &aThreadNetif):
     mTxFrame = static_cast<Frame *>(otPlatRadioGetTransmitBuffer(mNetif.GetInstance()));
 }
 
+void Mac::PrepareRadio()
+{
+    otPlatRadioSetCallbacks(mNetif.GetInstance(), ReceiveDoneTask, TransmitDoneTask);
```

Comment:
Why are these calls being assigned as dynamic callbacks rather than the fixed calls that exist currently?